meteor/client/ui/Settings/ConfigManifestSettings.tsx

```diff
@@ -552,53 +634,62 @@ export class ConfigManifestTable<
 												this.props.configPath,
 												this.props.object,
 												col,
-												`${baseAttribute}.${sortedIndices[i]}.${col.id}`
+												`${baseAttribute}.${sortedIndices[i]}.${col.id}`,
+												undefined,
+												undefined,
+												this.props.alternateObject
 											)}
 										</td>
 									))}
-									<td>
-										<button
-											className={ClassNames('btn btn-danger', {
-												'btn-tight': this.props.subPanel,
-											})}
-											onClick={() => this.removeRow(val._id, baseAttribute)}
-										>
-											<FontAwesomeIcon icon={faTrash} />
-										</button>
-									</td>
+									{!configEntry.disableRowManipulation && (
+										<td>
+											<button
+												className={ClassNames('btn btn-danger', {
+													'btn-tight': this.props.subPanel,
+												})}
+												onClick={() => this.removeRow(val._id, baseAttribute)}
+											>
+												<FontAwesomeIcon icon={faTrash} />
+											</button>
+										</td>
+									)}
 								</tr>
 							))}
 						</tbody>
 					</table>
 				</div>
-				<button
-					className={ClassNames('btn btn-primary', {
-						'btn-tight': this.props.subPanel,
-					})}
-					onClick={() => this.addRow(configEntry, baseAttribute)}
-				>
-					<FontAwesomeIcon icon={faPlus} />
-				</button>
-				<button
-					className={ClassNames('btn mlm btn-secondary', {
-						'btn-tight': this.props.subPanel,
-					})}
-					onClick={() => this.exportJSON(configEntry, vals)}
-				>
-					<FontAwesomeIcon icon={faDownload} />
-					&nbsp;{t('Export')}
-				</button>
-				<UploadButton
-					className={ClassNames('btn btn-secondary mls', {
-						'btn-tight': this.props.subPanel,
-					})}
-					accept="application/json,.json"
-					onChange={(e) => this.importJSON(e, configEntry, baseAttribute)}
-					key={this.state.uploadFileKey}
-				>
-					<FontAwesomeIcon icon={faUpload} />
-					&nbsp;{t('Import')}
-				</UploadButton>
+				{configEntry.disableRowManipulation === true || (
```

Comment:
`configEntry.disableRowManipulation === true` can be shortend to `configEntry.disableRowManipulation`

Reply:
Changed!